### PR TITLE
Different prompt

### PR DESCRIPTION
**Pull request by olegispe**
This basically allows the user to input a new fingerprint rather than exiting the program. 

It also includes just some spur-of-the-moment changes to `I18n` such as the french translations changing and need translation markers now being TODOs.

### CODE REVIEW

File: src/i18n.py [hunk 1]
```diff
@@ -209,8 +217,10 @@ def get(key, inline=False, color=None, end=Style.RESET_ALL):
 
             'websocket.closed': 'Websocket fechado',
             'websocket.opened': 'Websocket aberto',
-            'websocket.failed_collect': 'Failed to collect a webhook from PixelCanvas. [NEEDS TRANSLATION]',
-            'websocket.failed_connect': 'Failed to connect to the websocket. [NEEDS TRANSLATION]',
+            # TODO: NEEDS TRANSLATION
+            'websocket.failed_collect': 'Failed to collect a webhook from PixelCanvas.',
+            # TODO: NEEDS TRANSLATION
+            'websocket.failed_connect': 'Failed to connect to the websocket.',
```
Comment (4rm4) [inline src/i18n.py]: `            'websocket.failed_connect': 'Falha ao conectar ao Websocket..',`

File: src/i18n.py [hunk 2]
```diff
@@ -209,8 +217,10 @@ def get(key, inline=False, color=None, end=Style.RESET_ALL):
 
             'websocket.closed': 'Websocket fechado',
             'websocket.opened': 'Websocket aberto',
-            'websocket.failed_collect': 'Failed to collect a webhook from PixelCanvas. [NEEDS TRANSLATION]',
-            'websocket.failed_connect': 'Failed to connect to the websocket. [NEEDS TRANSLATION]',
+            # TODO: NEEDS TRANSLATION
+            'websocket.failed_collect': 'Failed to collect a webhook from PixelCanvas.',
```
Comment (4rm4) [inline src/i18n.py]: 'websocket.failed_collect': 'Falha ao coletar webhook do PixelCanvas.',

**Comment by Mismatchia** [comment]
We also need translations for `'error.cooldown': 'Existing cooldown not expired.`

File: src/i18n.py [hunk 3]
```diff
@@ -198,7 +202,11 @@ def get(key, inline=False, color=None, end=Style.RESET_ALL):
             'error.rate_limit': 'Oh não! Limite de tentativas excedido',
             'error.connection': 'Erro de conexão :(',
 
-            'paint.has_painted': 'Você pintou um pixel no pixelcanvas.io? y(sim)/n(não):',
+            # TODO: NEEDS TRANSLATION
+            'fingerprint_input': 'Please enter your new fingerprint: ',
+
+            # TODO: NEEDS TRANSLATION
+            'paint.has_painted': 'Have you painted a pixel in pixelcanvas.io? Yes (y), no (n) or change fingerprint (f):',
```
Comment (4rm4) [inline src/i18n.py]: 'paint.has_painted': 'Você pintou um pixel no pixelcanvas.io? Sim (y), Não (n) ou Mudar fingerprint (f):',

File: src/i18n.py [hunk 4]
```diff
@@ -198,7 +202,11 @@ def get(key, inline=False, color=None, end=Style.RESET_ALL):
             'error.rate_limit': 'Oh não! Limite de tentativas excedido',
             'error.connection': 'Erro de conexão :(',
 
-            'paint.has_painted': 'Você pintou um pixel no pixelcanvas.io? y(sim)/n(não):',
+            # TODO: NEEDS TRANSLATION
+            'fingerprint_input': 'Please enter your new fingerprint: ',
```
Comment (4rm4) [inline src/i18n.py]: 'fingerprint_input': 'Digite sua nova fingerprint: ',

**Comment by aud-tree** [comment]
Add @4rm4's translations, resolve the conflicts, and re-push this, and then I'll go ahead and merge it 🙂

**Comment by olegispe** [comment]
Reference #42.

**Pull request merged**